src/lib/connectors/repos/gauging-stations.js

```diff
@@ -1,10 +1,15 @@
 'use strict';
 const { GaugingStation } = require('../bookshelf');
 const helpers = require('./lib/helpers');
+const queries = require('./queries/gauging-stations');
+const raw = require('./lib/raw');
 
 const findOne = id =>
   helpers.findOne(GaugingStation, 'gaugingStationId', id);
 
+const findStationConditionsForId = gaugingStationId =>
```

Comment:
Could you kindly rename this function to `findStationConditionsById` (Replace `For` with `By`) - this isn't a big deal - it's just for consistency with the other functions.

Reply:
Sorry - actually could we call it `findLicenceConditionsByStationId` to make it clear which entity's ID is being passed here.